### PR TITLE
V3.8

### PR DESCRIPTION
**Pull request by ThDilos**
It's such a bold move to publish an update during exam season, while I'm close to failing all my modules, right?\
fox for life

---

**General QOL Changes:** 
Changed **MOST** Pehkui Size commands to use multiply and divide instead of set / reset\
What does this mean?
**You can use /scale command to adjust yourself to your desired size, for RP purpose or whatever. And your size retains, instead of being fixed to the default sizes.**
The only scale that still constantly resets to 1 is the entity reach.

[Fox Setting] Function:\
Changed the wiki link to the correct one

Active Skill [Hunt]\
Changed Contents:\
You can see the phase 1: preparation bar while in phase 2:hunt. This feature was accidentally removed in the last version.

Active Skill [Foxify]\
Changed Contents:
1. The [Sleepin mode] fox will spawn looking at your direction
2. Hold C against a crouching player to mount on them
3. Allow interaction to you, when you are riding something
4. Will constantly kill any errorous [Sleepin Mode] fox, not affecting game play
5. [Sleepin Mode] can now be disabled by yourself, by using /trigger foxtype set -1. But admin can still disable it for you, and their setting has higher priority.

Fixed Bugs:
1. [Sleepin Mode] Fox can now invincible against almost all damages, as intended. God I love boolean algebra.
2. Fixed Hat Fox blocking your view if you use newer version of Pehkui
3. Fixed you can hit Hat Fox if you use newer version of Pehkui
4. Because of this, the Hat Fox now has a hitbox thinner than a layer of snow. Thus it's very hard to hit em. I can't fix this :/

Passive Skill [Smol]\
Changed Contents:\
Removed Held Item Scaling of 0.7

Passive Skill [Timidity]\
Changed Contents:
1. You are now afraid of players with Human, Enderian, Blazeborn, Shulk Origin. Interacted by them will disable this fear. (Probably)
2. For disabled fear, added a received damage threshold of >=3 for you to fear them again.
3. You won't fear the wolves tamed by players with this origin. (Pet Wolfie for Bold Foxie)

Passive Skill [Foxiality]\
Changed Contents:
1. Shift + Right Click to force a fox to sit by removing their brain (NoAI:1b).
2. Do it again to un-sit them
3. If they get damaged, they automatically un-sit

Passive Skill [Fluffy]\
Changed Contents:
1. Trigger for shaking is reworked, you can only trigger shaking when you are not moving.
9. Evaporation reworked, you drain your wetness / snowness when you are moving (unless sneaking).
10. If the wetness / snowness is drained below a certain level, you no longer can trigger shaking.
11. When you can trigger shakes, you will emit particles when moving (water and snow).

**TL;DR: Stand still to shake, keep running to dry out, evaporation becomes more relevant.**

Hidden Skill [Thtrackers]\
Changed Contents:
1. Automatic foxie tag clearing if you change to another origin, but will not work if you do /power clear.
5. Automatically give you back foxie tag if you do not have one

2 New Non-Player Skills:
1. [fox/thforce_sit]: Unsit the fox if taking damage. Self-removed afterwards
2. [fox/wolf_collar]: Give you a 10 ticks windows to tame the wolf everytime a bone is used. If success: Self-remove and give the wolf "foxie_tamed" tag. If windows passed: Self-remove

---

Special thanks to people on the origin discord, they told me most of the bugs occured. And suggested some of the new contents above.

### CODE REVIEW

**Comment by ThDilos** [comment]
I love when ThDilos says "It's Merging time!" and merged all over the place
